Create README.md (#2)
pantheon-systems/pantheon-content-publisher-drupal: README.md

# Pantheon Content Publisher for Drupal

> Pantheon content publisher module allows you to integrate content created in Google Docs to your Drupal website.

Reviewer: ```suggestion
The Pantheon Content Publisher module allows you to integrate content created in Google Docs to your Drupal website.
```

> Refer the [documentation](https://pcc.pantheon.io/pantheon-content-publisher-for-drupal) for more information about installation and configuration of module.

Author: @jazzsequence Does it looks good?

-  A Google Workspace Account
-  Drupal 10+ site with PHP 8.2+

> -  For sites hosted on Pantheon , enable Pantheon Search and configure search version in pantheon.yml. Follow the instructions [here](https://docs.pantheon.io/guides/solr-drupal/solr-drupal#enable-at-the-site-level)

Reviewer: ```suggestion
-  For sites hosted on Pantheon, enable [Pantheon Search](https://docs.pantheon.io/solr) and configure the search version in your `pantheon.yml`. Follow the instructions [here](https://docs.pantheon.io/guides/solr-drupal/solr-drupal#enable-at-the-site-level)
```

-  PCC CLI (Pantheon Content Publisher Command Line Tool)
  

>    Directions to install the CLI are [here](https://pcc.pantheon.io/pcc-cli-setup).

Reviewer: ```suggestion
-  PCC CLI (Pantheon Content Publisher Command Line Tool). Instructions to install the CLI are [available here](https://pcc.pantheon.io/pcc-cli-setup).
```

## Installation

> To install this module via composer, run the following command

Reviewer: ```suggestion
To install this module via Composer, run the following command
```

```
composer require drupal/pantheon_content_publisher:"^1.0"

```
The module requires a backend Search API plugin. For sites hosted on Pantheon, it is recommended to use the Search API Pantheon module.
To install Search API Pantheon Module, run the following command
```
composer require drupal/search_api_pantheon:^8

```

> And then push the composer.json and composer.lock files to your Pantheon environment.

Reviewer: ```suggestion
After installing the Content Publisher and Search API Pantheon modules locally with Composer, push the `composer.json` and `composer.lock` files to your Pantheon environment.
```

> Refer the documentation for more information about installation and configuration of module [pcc.pantheon.io/pantheon-content-publisher-for-drupal](https://pcc.pantheon.io/pantheon-content-publisher-for-drupal)

Reviewer: This might be better placed higher in the readme.
